pkg/test/setup/setup.go

```diff
@@ -91,7 +91,6 @@ func modules(flags []string) func() {
 
 	return test.SetupModules(
 		flags,
-		repo,
```

Comment:
Why do you move `repo` to the last?

Reply:
The order of sequence should be equal to https://github.com/apache/skywalking-banyandb/blob/main/banyand/internal/cmd/standalone.go#L81-L92

Reply:
ok